python/catkin/builder.py

```diff
@@ -287,17 +288,8 @@ def get_multiarch():
 
 def get_python_install_dir():
     # this function returns the same value as the CMake variable PYTHON_INSTALL_DIR from catkin/cmake/python.cmake
```

Comment:
What if the tool is invoked with a different python executable than the one found in `PYTHON_EXECUTABLE` in cmake?

I can imagine, that if the tool is using Python3 and then the CMake code `find_package(PythonInterp)` finds Python2, the code will be installed to something like `<prefix>/lib/python2.7/site-packages` but the path `<prefix>/lib/python3.5/site-packages` would be added to the `PYTHONPATH` using the setup file which is generated elsewhere in this file.

Or maybe this function is really only about getting `site-packages` versus `dist-packages`?